<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DiagramAddSymbol</class>
 <widget class="QWidget" name="DiagramAddSymbol">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>410</width>
    <height>292</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Add Symbol to Diagram</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_3">
   <item>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="QLabel" name="label">
         <property name="text">
          <string>Symbols in Diagram</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QListWidget" name="lwSymbols"/>
       </item>
       <item>
        <widget class="QPushButton" name="pbPlace">
         <property name="text">
          <string>Place a copy of selected symbol</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QLabel" name="label_2">
         <property name="text">
          <string>Add from file</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="Gui::FileChooser" name="fcFile" native="true"/>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>Gui::FileChooser</class>
   <extends>QWidget</extends>
   <header>Gui/FileDialog.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
